### PR TITLE
Entities mni

### PR DESCRIPTION
**Pull request by Matoosh**
Aaaarrrrgggghhhh! Ahoy, comrades! I made some code. It would be nice if you check it.

### CODE REVIEW

File: core/src/main/java/app/majime/core/sample/Sample.java [hunk 1]
```diff
@@ -19,20 +19,23 @@
...
-    private int status;
+    @Column(nullable = false)
+    private Long status;
 
-    private int batch_id;
+    @Column(nullable = false)
+    private Long batch_id;
 
-    private int user_id;
+    @Column(nullable = false)
```
Comment (Pirer24) [inline core/src/main/java/app/majime/core/sample/Sample.java:34]:
```suggestion
    @NotNull
```
It's deprecated!! Plz Boy, check it out

File: core/src/main/java/app/majime/core/resultStatusHistory/ResultStatusHistory.java [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+
+    @Column(length = 50, nullable = false)
     private String oldValue;
+
+    @Column(length = 50, nullable = false)
     private String newValue;
-    private long resultId;
-    private long userId;
+
+    @Column(nullable = false)
```
Comment (Pirer24) [inline core/src/main/java/app/majime/core/resultStatusHistory/ResultStatusHistory.java:27]:
```suggestion
    @NotNull
```
AGAIN!
It's deprecated!! Plz Boy, check it out

File: core/src/main/java/app/majime/core/sampleLab/SampleLab.java [hunk 3]
```diff
@@ -1,8 +1,28 @@
...
+    @SequenceGenerator(name="sample_lab_seq", sequenceName="sample_lab_id_seq", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sample_lab_seq")
+    private Long id;
+
+    @Column
     private int quantity;
-    private long sampleId;
-    private long labId;
+
+    @Column
```
Comment (Pirer24) [inline core/src/main/java/app/majime/core/sampleLab/SampleLab.java:23]: Single @Column ? For what? 
```suggestion
```

**Review by Pirer24 — COMMENTED** [review]
Check my review

**Comment by Matoosh** [comment]
Boiii @Pirer24, 
I made changes as you recommended.

**Pull request merged**